src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
```

Comment:
Splitting by semicolon ';' may lead to empty strings in the 'declarations' array, especially given the provided example with multiple semicolons. Consider filtering out empty strings to make the code more robust.